academy/0.0-B9lab-Blockchains/1_blockchain.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Blockchain 101"
+order: 3
+description: A universe to discover
+tag: fast-track
+---
+
+This first module demystifies the foundations of blockchain technology. This background information will be important as you progress through the course. 
+
+The modules include learning materials for your studies, as well as examples and exercises for you to practice. You will also find freely accessible resources from other sites to help you dive deeper into the subject matter at hand.
```

Comment:
The modules or the module? If you are referring to the modules in the whole Academy training, I would recommend: "The chapters in the Cosmos Developer Academy..." or "The Cosmos Developer Academy". If only the Introduction is meant, then I would recommend changing it to: "The chapter".

Reply:
I'm talking about the _terminology_ (chapter? section?) that refers to everything in `0.0-B9lab-Blockchains` which is 5 pages of ambient knowledge that everyone should know before they attempt to unpack Cosmos.

Reply:
Then I would go with chapter. ;-)